packages/channels/botpress-channel-web/src/db.js

```diff
@@ -132,6 +144,7 @@ module.exports = knex => {
       .insert({
         userId,
         created_on: helpers(knex).date.now(),
+        last_heard_on: helpers(knex).date.now(),
```

Comment:
`last_hear_on` is meant to mean "last heard from the user". In this case it's the bot speaking

Reply:
Wait, is it? It's `createConversation` method. 
I've made this change because the _recent conversation_ is looked up by this field. Before this change for the newest convo the field was `null`. So it wouldn't be found as the newest convo when ordering by this field DESC. Am I wrong?

Reply:
I don't want to merge before we resolve this question.

Reply:
I *think* the way I saw that back then was that if the user didn't _reply_ to the bot, it is not considered as "recent" because there hasn't actually been any exchange of messages, right? When looking at it this way, you want to actually bring back to the user the last exchange he had with the bot.

Reply:
Ah, that makes sense (though then we need to exclude NULL entries from this query). Will update

Reply:
Hm, I think there's actually a problem wtih our tests. 
If we don't init this field on the original user message there's a gap between the convo is created and the message is inserted (at which point the convo last_heard_on is bumped).
When we make a sttress test with messages going rapidly from the same user (as with our stress test) there'a a dozen of message arriving almost simultaiously. At this point a dozen new convos is created because messages arrive faster than  the insert convo - insert message - update convo's last_heard_on cycle happens. So I will update it so that the convo is initialized with last_heard_on when it originates from the user (but not when from the bot)